history/history-network.md

```diff
@@ -412,3 +427,49 @@ flowchart LR
     ExecutionBlockProof --> Proof2([verify_merkle_multiproof])
     beaconBlockRoot --> Proof2 --> block_hash
 ```
+
+#### BlockProofHistoricalSummaries
+
+The `BlockProofHistoricalSummaries` is an SSZ container which holds two Merkle proofs as specified in the [SSZ Merke proofs specification](https://github.com/ethereum/consensus-specs/blob/dev/ssz/merkle-proofs.md#merkle-multiproofs):
+- `BeaconBlockProofHistoricalSummaries`
+- `ExecutionBlockProof`
+
+Additionally the SSZ container holds a `BeaconBlock` hash_tree_root and a slot.
+
+This chain of two proofs allows for verifying that an EL `BlockHeader` is part of the canonical chain. The only requirement is having access to the beacon chain `historical_summaries`.
+The `historical_summaries` is a [`BeaconState` field](https://github.com/ethereum/consensus-specs/blob/dev/specs/capella/beacon-chain.md#beaconstate) that was introduced since the Capella fork. It gets updated every period (8192 slots). The `BlockProofHistoricalSummaries` MUST be used to verify blocks from the Capella fork onwards.
+
+The Portal beacon network [provides access](./beacon-chain/beacon-network.md#historicalsummaries) to an up to date `historical_summaries` object.
```

Comment:
This can lead to confusion that the `Portal Beacon Network` is the only option to get these.

I think we should either
- not mention the Portal Beacon Network
- or mention multiple sources implementations could get this data, for example an EL client wouldn't run the Portal Beacon Network as it already has access to this data from their CL client.

The reason I mention this as it is a question I had to answer multiple times to EL layer teams and is consistently a cause of confusion

Reply:
Perhaps I can alter it to:

"The `historical_summaries` can be taken from the CL `BeaconState` (Link to beacon API endpoint) or the Portal beacon network also provides access to an up to date `historical_summaries` object."

And hopefully in a secondary PR I can add a specific API endpoint for it :)

Reply:
This sounds like a good solution to me because then it is clear you aren't forced to use Portal beacon but it is 1 of 2 options, which should prevent confusion

Reply:
adjusted in https://github.com/ethereum/portal-network-specs/pull/291/commits/ec0200cb52c43d57973b0cf40d242757def95938